homeassistant/components/device_tracker/quantum_gateway.py

```diff
@@ -14,14 +14,16 @@
 from homeassistant.const import (CONF_HOST, CONF_PASSWORD)
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['quantum-gateway==0.0.3']
+REQUIREMENTS = ['quantum-gateway==0.0.5']
 
 _LOGGER = logging.getLogger(__name__)
 
 DEFAULT_HOST = 'myfiosgateway.com'
+USE_HTTPS = 'use_https'
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Optional(CONF_HOST, default=DEFAULT_HOST): cv.string,
```

Comment:
Use CONF_SSL which is used for this use case in many other components: 
https://github.com/home-assistant/home-assistant/blob/127c55e0c196dc6ffd321c1ad28b533f636faf79/homeassistant/const.py#L136